recipe/meta.yaml

```diff
@@ -28,7 +28,7 @@ requirements:
         - g2clib
         - hdfeos2
         - hdfeos5
-        - gcc
+        - gcc  # [osx]
```

Comment:
Yeah, I think we still have to use this on Linux if we want a `gfortran` compiler at present.